### PR TITLE
Add ID to Book Call Button on About Us Page

### PR DESCRIPTION
**Pull request by jessica-lau**
### What does this pull request aim to achieve? 💯

Add an id to the Book Call button on the About Us page to help simplify cypress test script.

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

**Review by ddveloper — COMMENTED** [review]
please define in `gtm-tags.js` and use `getGTM()` function, same for "AU_PR" in line 425

**Comment by jessica-lau** [comment]
> please define in `gtm-tags.js` and use `getGTM()` function, same for "AU_PR" in line 425

@ddveloper added IDs to gtm-tags.js

**Review by TimeaKovacs86 — DISMISSED** [review]
Looks good to me! When it is merged, I will update the tests for the About Us page! Thank you @jessica-lau. If you find it ok, could you please merge @ddveloper ? Thank you !

File: client/src/pages/AboutUs.js [hunk 1]
```diff
@@ -396,18 +399,21 @@ const AboutUs = () => {
           {t("companyLogoInfo")}
         </p>
         <h3>{t("lifetimeSupporters")}</h3>
-        <SupportersLogosContainer id="AU_LS">
+        <SupportersLogosContainer id={getGTM("lifetimeSupporters")}>
```
Comment (galeza) [inline client/src/pages/AboutUs.js]: @jessica-lau @ddveloper 
There's no need to add GTM tags for these containers they are not tracked. We need only ID for testing purposes..
Also Press release button is not tracked as far as I remember. 
Please clarify this with @FedericaZn

**Review by galeza — CHANGES_REQUESTED** [review]

**Comment by galeza** [comment]
Hi @FedericaZn 
Do you track SupportersLogosContainers (on About us page?) and need GTM tags there? 
I don't think so. Please confirm.

File: client/src/pages/AboutUs.js (hunk 1 above)
Comment (jessica-lau) [reply client/src/pages/AboutUs.js]: @galeza sorry I wasn't aware. I'll remove the GTM tags for the containers after @FedericaZn confirms and leave the container IDs in the aboutUS.js as is. Thinking about it, it does make sense that the containers and press release button wouldn't be tracked.

Comment (ddveloper) [reply client/src/pages/AboutUs.js:402]: my bad 😅

**Review by galeza — COMMENTED** [review]
Hi @jessica-lau
I clarified GTM tags with Federica:
1. We don't need GTM tags for SupportersLogosContainers - present and lifetime
2. Keep GTM tag for Go to Press Kit
3. Keep GTM tag for Book a call

Thanks

**Comment by jessica-lau** [comment]
> Hi @jessica-lau
> I clarified GTM tags with Federica:
> 
> 1. We don't need GTM tags for SupportersLogosContainers - present and lifetime
> 2. Keep GTM tag for Go to Press Kit
> 3. Keep GTM tag for Book a call
> 
> Thanks

Great, thanks @galeza. I made the updates.

**Review by TimeaKovacs86 — APPROVED** [review]
I think this is ok now... if I am not mistaken and there are no other suggestions. Thank you @jessica-lau When this PR get the approval and merge I will update the tests for the About US page then.

**Comment by TimeaKovacs86** [comment]
Hi everyone! Do we have any updates on this PR? Can it be merged or is there any changes requested?

**Review by galeza — APPROVED** [review]
LGTM

**Comment by galeza** [comment]
Hi @ddveloper 
Could we merge this PR?
Thanks

**Comment by ddveloper** [comment]
sure @galeza 👍

**Comment by TimeaKovacs86** [comment]
Great! Thanks!

**Comment by ghost** [comment]
Element's ID is present.

**Pull request merged**